sdk/keyvault/azure-security-keyvault-keys/src/cryptography/cryptography_client.cpp

```diff
@@ -58,15 +57,43 @@ inline std::vector<uint8_t> CreateDigest(
 }
 } // namespace
 
+Request CryptographyClient::CreateRequest(
+    HttpMethod method,
+    std::vector<std::string> const& path,
+    Azure::Core::IO::BodyStream* content) const
+{
+  return Azure::Security::KeyVault::_detail::KeyVaultKeysCommonRequest::CreateRequest(
+      m_keyId, m_apiVersion, method, path, content);
+}
+
+std::unique_ptr<Azure::Core::Http::RawResponse> CryptographyClient::SendCryptoRequest(
+    std::vector<std::string> const& path,
+    std::string const& payload,
+    Azure::Core::Context const& context) const
+{
+  // Payload for the request
+  Azure::Core::IO::MemoryBodyStream payloadStream(
+      reinterpret_cast<const uint8_t*>(payload.data()), payload.size());
+
+  // Request and settings
+  auto request = CreateRequest(HttpMethod::Post, path, &payloadStream);
+  request.SetHeader(HttpShared::ContentType, HttpShared::ApplicationJson);
+  request.SetHeader(HttpShared::Accept, HttpShared::ApplicationJson);
```

Comment:
I don't see where ANY of these operations want Content-Type or Accept headers.
I also see some other request values (like digest, add, tag, & value) that these functions don't seem to support at all. Do all languages just ignore these?
For reference: https://docs.microsoft.com/en-us/rest/api/keyvault/encrypt/encrypt

Reply:
The REST doc is not as good as it should.

I tried removing the content-type header and the service return an error `BadRequest` without it.

What I think is that this header is an standard header which is usually automatically added by an HTTP client.  Because, at least on PostMan, if we send any payload as JSON, the client adds the content-type header automatically.

So, I believe the rest docs are implicitly expecting the client to add the header whenever there's a json payload

Reply:
> I also see some other request values (like digest, add, tag, & value)

The client is helping customers to build valid requests only by exposing EncryptParameters model and a factory methods for each type of algorithm.
For example, for Encrypt, the usage of iv/aad depends on the algorithm used.

Reply:
I see; I just found this: https://docs.microsoft.com/en-us/azure/key-vault/general/authentication-requests-and-responses#request-body